### ISSUE TITLE
Fails to link `ricochet-refresh` due to missing symbol related to `abseil`

### ISSUE THREAD

**Issue by odiferousmint** [blueprint-freespeech/ricochet-refresh#178]
Hello.

When trying to compile, I get the following error:

```
[100%] Linking CXX executable ricochet-refresh/ricochet-refresh
/usr/bin/ld: ../libtego/libtego.a(ControlChannel.pb.cc.o): undefined reference to symbol '_ZN4absl12lts_2023080212log_internal17MakeCheckOpStringIllEEPNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEET_T0_PKc'
/usr/bin/ld: /usr/lib/libabsl_log_internal_check_op.so.2308.0.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [ricochet-refresh/CMakeFiles/ricochet-refresh.dir/build.make:433: ricochet-refresh/ricochet-refresh/ricochet-refresh] Error 1
make[1]: *** [CMakeFiles/Makefile2:305: ricochet-refresh/CMakeFiles/ricochet-refresh.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

Any ideas?

**Comment by pospeselr** [comment]
The abseil dependency is coming from protobuf which was added in the 3.22 series (which is why we build against the 3.21 series in the official releases: https://github.com/blueprint-freespeech/ricochet-build/blob/6cec75b70cfdd6d237ceb2d66328960afb33b073/rbm.conf#L66)

For a local build you should be able to install the abseil the abseil dev package for your distro.

**Comment by milahu** [comment]
fixed in #182
